packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -286,6 +287,13 @@ function TokenBalance({
   tokenSymbolOverride?: string
 }) {
   const { l1, l2 } = useNetworksAndSigners()
+  const isL1 = on === NetworkType.l1
+  const chain = isL1 ? l1.network : l2.network
+
+  const isERC20BridgeToken = (
+    token: ERC20BridgeToken | NativeCurrencyErc20 | null
+  ): token is ERC20BridgeToken =>
+    token !== null && !token.hasOwnProperty('isCustom')
```

Comment:
Wouldn't `token !== null && !token.isCustom` be sufficient?

Reply:
yea i wanted to
but can't do `token.isCustom` because `ERC20BridgeToken` doesn't have `isCustom` :(
<img width="521" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/12451f1f-1775-47e9-ad21-599fdbaccbb1">